src/it/resources-noncompilable/com/google/checkstyle/test/chapter5naming/rule528typevariablenames/InputRecordTypeParameterName.java

```diff
@@ -1,54 +1,52 @@
 // non-compiled with javac: Compilable with Java17
+
 package com.puppycrawl.tools.checkstyle.checks.naming.recordtypeparametername;
 
 import java.io.Serializable;
 import java.util.LinkedHashMap;
 import org.w3c.dom.Node;
 
-/* Config:
- * pattern = "(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)"
- *
- */
+/** some javadoc. Config: pattern = "(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)" */
 public record InputRecordTypeParameterName<t>(Integer x, String str) {
```

Comment:
record didn't had proper indentation, maybe I forgot to format this file in previous issue. so diff has become huge, reviewing it without whitespace would be easy and also moved extra records inside main record.